uber/config.py

```diff
@@ -231,7 +231,7 @@ def __getattr__(self, name):
             else:
                 return sa.localized_now() > date_setting
         elif name.startswith('HAS_') and name.endswith('_ACCESS'):
-            return getattr(c, name.split('_')[1]) in sa.AdminAccount.access_set()
+            return getattr(c, '_'.join(name.split('_')[1:-1])) in sa.AdminAccount.access_set()
```

Comment:
Previously ``c.HAS_REG_AT_CON_ACCESS`` was broken because the old version assumed that access level names didn't have underscores.